src/xenia/config.h

```diff
@@ -0,0 +1,52 @@
+#ifndef XENIA_CONFIG_H_
+#define XENIA_CONFIG_H_
+#include <cpptoml/include/cpptoml.h>
+#include <string>
+
+class CmdVar {
+ public:
+  CmdVar(const char* name, const char* defaultValue, const char* description);
```

Comment:
Add a `flags` parameter to this constructor (default 0). We can define CVar flags such as one that says this CVar cannot be modified at runtime.
We can also add a CVar changed callback function parameter (once again, default NULL).